src/audio/base_fw.c

```diff
@@ -13,6 +13,7 @@
 #include <sof/lib/cpu.h>
 #include <rtos/init.h>
 #include <platform/lib/clk.h>
+#include <intel_adsp_hda.h>
```

Comment:
This is problematic. This file is not Intel specific so any Intel-specifc code and headers need to be ifdeffed (ideally put to a different file, but that is a bigger reworked and not needed now). But to not break builds, thi needs ifdef guards.